doc/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,56 @@
+## Why triage?
+
+Triage is a process of sanity checking and prioritizing issues. The term is adopted from medical contexts where patients are briefly examined to decide whether they need immediate care, or if their issue can wait until patients at greater risk can be handled first.
+Software issue triage is not life-or-death, but it does adress the same fundamental problem, which is that there is a limited amount of attention and skill that can be brought to bear on issues at any given time, and some issues have higher *impact* than others.
+
+If issues are properly triaged, contributors with time available can focus on getting things done instead of searching through issues for something to work on and fix.
+
+## How does triage work?
+Look at new issues or revisit old issues after a while and apply various labels:
+1. Characterize as a bug report or a feature suggestion
+2. if bug report:
+    1. label with priority
+    2. check for validity
+3. if feature suggestion:
+    1. evaluate for validity
+4. in either case, also apply labels indicating what areas of the game it impacts, i.e. content/json vs features/code, items vs monsters vs translation etc.
+
+### Is this a bug report or a feature request?
+Sometimes this is simple, is the game crashing, is it losing save data? If so it's definitely a bug.
+Likewise, new content or new ways to interact with the game are always feature requests.
+Other times this is a surprisingly tricky question. "the game isn't working like I expect it to" doesn't actually mean the game is doing anything wrong, and even if it is (i.e. it sets up an expectation but doesn't meet it), the problem might be where the expectation is set rather than the point where the expectation is (not) met.
```

Comment:
```suggestion
Other times this is a surprisingly tricky question. "The game isn't working like I expect it to" doesn't actually mean the game is doing anything wrong, and even if it is (i.e. it sets up an expectation but doesn't meet it), the problem might be where the expectation is set rather than the point where the expectation is (not) met.
```